### PR TITLE
add stars

### PR DESCRIPTION
**Pull request by 1v43k**
- [DEMO LINK](https://1v43k.github.io/layout_stars/)
- [TEST REPORT LINK](https://1v43k.github.io/layout_stars/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  background-image: var(--active-star);
+}
+
+.stars--4 :nth-child(-n+4) {
+  background-image: var(--active-star);
+}
+
+.stars--5 :nth-child(-n+5) {
+  background-image: var(--active-star);
+}
```
Comment (IvanFesenko) [inline src/style.css:43]: You can use comma
```suggestion
.stars--1 :nth-child(-n+1),
.stars--2 :nth-child(-n+2),
.stars--3 :nth-child(-n+3),
.stars--4 :nth-child(-n+4),
.stars--5 :nth-child(-n+5) {
  background-image: var(--active-star);
}
```

**Review by IvanFesenko — CHANGES_REQUESTED** [review]
Check comment

**Comment by 1v43k** [comment]
> Your link isn't work

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,43 @@
+:root {
+  --star: url(./images/star.svg);
+  --active-star: url(./images/star-active.svg);
+}
+
+* {
+  margin: 0;
```
Comment (VitaliyBondarenko1982) [inline src/style.css:7]: Redundant to make margin: 0 for all selectors.
Make just where it necessary. In this case just for div.

File: src/style.css (hunk 1 above)
Comment (VitaliyBondarenko1982) [reply src/style.css:43]: Please fix this comment.

**Review by VitaliyBondarenko1982 — COMMENTED** [review]
PR closed.

**Comment by 1v43k** [comment]
> PR closed. Why did you close PR?

In this PR my npm module didn't work correctly, since I didn't success to delete and reinstall it, I decided to close it and create a new one  #2522

File: src/style.css (hunk 2 above)
Comment (VitaliyBondarenko1982) [reply src/style.css:7]: UPD: * just for body

**Pull request closed without merging**